Dockerfile.alpine

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.7
+
+RUN apk add -U --no-cache ca-certificates
```

Comment:
only fetch apk cache once. Just remove `-U` flag. See

https://github.com/drone-plugins/drone-base/commit/b344bc7d287841780932bbc287cddf66760e6516#diff-3254677a7917c6c01f55212f86c57fbf

Reply:
Sorry didn't see this in time.

Reply:
https://github.com/drone/drone-cli/pull/84